services/dynamic-sidecar/tests/unit/test_api_containers.py

```diff
@@ -53,7 +67,9 @@ def compose_spec(dynamic_sidecar_network_name: str) -> str:
             "services": {
                 "first-box": {
                     "image": "busybox",
-                    "networks": [dynamic_sidecar_network_name],
+                    "networks": [
+                        dynamic_sidecar_network_name,
```

Comment:
This is not the first time I see it, but what is the advantage in adding this comma?

Reply:
I add it to enforce black to create multiline so the json-like structure looks like yaml. That would be
```yaml
services:
       first-box:
              image: busybox
              networks:
                   - dynamic_sidecar_ntework_name
```

It helps me seeing better all sections since we are used to yaml more than json for the compose specs.